### PR TITLE
Added a FLAC file to the Sound example to cover all three supported formats.

### PR DESCRIPTION
**Pull request by eXpl0it3r**
Now that we're down to three supported formats, we should also "test" all of them in the example.

### CODE REVIEW

File: examples/sound/Sound.cpp [hunk 1]
```diff
@@ -87,8 +88,11 @@ int main()
     // Play a sound
     playSound();
 
-    // Play a music
-    playMusic();
+    // Play music from an ogg file
+    playMusic("orchestral.ogg");
+    
+    // Play music from an flac file
+    playMusic("song.flac");
```
Comment (LaurentGomila) [inline examples/sound/Sound.cpp]: Where does it come from? Is it free? And 1 MB, it's a lot, when I released SFML 2.0 I remember that I replaced all the resources to reduce the total size of SDKs. And it's also better if the music is not too long for this example.

File: examples/sound/Sound.cpp [hunk 2]
```diff
@@ -46,15 +47,15 @@ void playSound()
 /// Play a music
 ///
 ////////////////////////////////////////////////////////////
-void playMusic()
+void playMusic(const sf::String& filename)
```
Comment (LaurentGomila) [inline examples/sound/Sound.cpp]: ```sf::Music::openFromFile``` takes a ```std::string```, so here you're just giving the false impression that ```sf::String``` (and thus Unicode) is supported :p

File: examples/sound/Sound.cpp (hunk 1 above)
Comment (eXpl0it3r) [inline examples/sound/Sound.cpp]: It's a sample from OpenGameArt and listed under pulic domain. I can try to find a shorter one.

File: examples/sound/Sound.cpp (hunk 2 above)
Comment (eXpl0it3r) [inline examples/sound/Sound.cpp]: Right, I wanted to check this, but then was in a hurry...

**Comment by eXpl0it3r** [comment]
Currently doesn't play due to #814 otherwise I adjusted the PR according to the comments ([FLAC reference](http://opengameart.org/content/magic-spell-sfx)).

File: examples/sound/Sound.cpp [hunk 3]
```diff
@@ -3,9 +3,10 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <SFML/Audio.hpp>
+
```
Comment (LaurentGomila) [inline examples/sound/Sound.cpp]: Empty line :)

**Comment by eXpl0it3r** [comment]
Fixed.

**Comment by zsbzsb** [comment]
:+1:

**Comment by eXpl0it3r** [comment]
Can we add this to 2.3?

**Comment by mantognini** [comment]
Why not ;)

Code is simple and clean. Good for me.

**Comment by Bromeon** [comment]
This should be rebased onto master (maybe even after the merge of `bugfix/findsfml_fixes`). I can't build this version with CMake because of the wrong FreeType paths.

**Comment by eXpl0it3r** [comment]
I wonder how you build things, because it works perfectly fine for me, but I'll certainly rebase it. :smile:

**Comment by Bromeon** [comment]
Funny :grinning: In the log graph, it was visible that `feature/example_flac` was based on a commit before the Freetype change. Maybe an old version? Although I did pull...

In any case, it works now :+1:

**Pull request merged**